contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -148,6 +150,7 @@ contract TrueCreditAgency is UpgradeableClaimable {
         require(isPoolAllowed[pool], "TrueCreditAgency: The pool is not whitelisted for borrowing");
         uint8 oldScore = creditScore[pool][msg.sender];
         uint8 newScore = creditOracle.getScore(msg.sender);
+        nextRepaymentTerm[pool][msg.sender] = block.timestamp.add(REPAYMENT_PERIOD);
```

Comment:
No good. If someone has some debt already, he can push the moment of repayment forward, without repaying, but with just borrowing more. It cannot work like that.